app/assets/javascripts/activeadmin_addons/select2.js

```diff
@@ -16,24 +16,33 @@ $(function() {
     setupAjaxSearch();
     setupTags();
 
+    function getSelectOptions($el) {
+      var base = {
+            tags : $el.data('collection'),
```

Comment:
Expected indentation of 8 spaces but found 12                    indent<br>Extra space after key 'tags'                                     key-spacing